index.bs

```diff
@@ -52,6 +52,12 @@ dictionary StorageBucketOptions {
 </xmp>
 
 <h3 id="storage-bucket-open">Creating a bucket</h3>
+
+When not null, {{StorageBucketOptions/durability}} is a hint to the user agent
+specifying the desired {{StorageBucket/durability}}. The user agent MAY
```

Comment:
Is it standard to make these all caps? I've never noticed this pattern before but my spec knowledge is lacking.

Reply:
Historically more w3c specs used all-caps for these. https://infra.spec.whatwg.org/#conformance now says that using lower-case is fine as well, despite that being a violation of some RFC (although not sure it is linking to the right RFC).

Reply:
Interesting, thanks for the historical context! Will leave it up to Evan then.